docs/README.md

```diff
@@ -67,14 +67,14 @@ following the instructions.
         $ cd /path/to/marathon-gh-pages
         $ git checkout gh-pages
 
-3. Copy the contents of the "docs" directory in master to the root of your
+3. Check out the appropriate release branch, then copy the contents of the "docs" directory in master to the root of your
```

Comment:
Step two (2) is redundant after this change. I think this point should be splited and replace step two.